packages/flame_riverpod/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: flame_riverpod
+description: Helpers for using Riverpod - a reactive caching and data-binding framework, 
+  in conjunction with Flame. The ComponentRef is used as an intermediary between Flame and
+  the Widget tree in order to provide state to Flame Components.
+version: 4.0.0+2
```

Comment:
why start on this specific version?

Reply:
This is an existing package, and that version is the current live version